nbs/examples/Large_Datasets_Tutorial.ipynb

```diff
@@ -0,0 +1,726 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #9.    files_list = [f"data/{dir}" for dir in os.listdir('./data')]
</pre><p>Please also use the tempdir here</p>

<pre class="ql-syntax" spellcheck="false">files_list = [f"data/{dir}" for dir in os.listdir(tmpdir.name)]
</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Nixtla/neuralforecast/pull/1074/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>

Reply:
sounds good - just flagging I've set the file list to be "(tmpdir.name)/{dir}" instead